app/controllers/matches_controller.rb

```diff
@@ -1,5 +1,6 @@
 class MatchesController < ApplicationController
   before_action :set_match, only: [:show, :edit, :update, :update_points, :edit_results, :update_results, :destroy, :add_game_result, :remove_game_result]
+  authorize_resource only: [:edit]
```

Comment:
Why only edit?

Reply:
US 332 only specifies to restrict editing

Reply:
then create a ticket? maybe ask @PoJo93